### PR TITLE
documentation: updated README with new workshop details

### PR DESCRIPTION
**Pull request by rjjaegeraws**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,6 +166,11 @@ This summarizes the installation process, the full installation document can be
 
 - Accelerator Prescriptive Architecture Sample [diagrams](./docs/architectures/AWS_Diagrams_Account_Network_VPC.md)
 
+### - Workshops
+
+- **The ASEA Administrator Immersion Day** workshop is focused on helping administrators who will be administering the landing zone understand how they can design, build and operate the components in ASEA. Click [here](https://catalog.us-east-1.prod.workshops.aws/v2/workshops/f3ed5d0f-d2f1-47e8-a305-168da9179aaa/en-US/sea-administrators) for an overview of the topics covered.
```
Comment (Brian969) [inline README.md:171]: Suggested consistency tweaks:
- ASEA has not been used or defined on this page.  Suggest changing to: "~~The ASEA~~ Accelerator Administrator Immersion Day".
- Similarly, landing zone is not used on this page (at least in this context).  Suggest changing "administering the ~~landing zone~~ understand" to "administering the **Accelerator environment to** understand".

File: README.md [hunk 2]
```diff
@@ -166,6 +166,11 @@ This summarizes the installation process, the full installation document can be
...
+- **The ASEA Project Team Immersion Day** workshop is focused on helping project teams understand what it means to operate within an ASEA managed environment. Click [here](https://catalog.us-east-1.prod.workshops.aws/v2/workshops/f3ed5d0f-d2f1-47e8-a305-168da9179aaa/en-US/sea-members) for an overview of the topics covered.
```
Comment (Brian969) [inline README.md:172]: - change ASEA to Accelerator per above
- would "Workload/Application Team" be clearer instead of "Project"?

**Comment by Brian969** [comment]
Love it and a long time coming. These Immersions days are a HUGE step forward!

File: README.md (hunk 2 above)
Comment (rjjaegeraws) [reply README.md]: - I updated from ASEA to Accelerator
- Also changed it to Workload/Application team for this context (remains Project on the workshop site)

**Review by Brian969 — APPROVED** [review]
LGTM

**Pull request merged**